src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0 auto;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.main {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  padding-top: 10px;
+
+}
+
+.list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: end;
+  height: 100%;
```

Comment:
overkill and if u will remove it nothing will change

Reply:
![image](https://user-images.githubusercontent.com/98764641/200609407-4797b273-bd38-46aa-8b1e-c33d8de4bac5.png)
Fix it